docs/events.md

```diff
@@ -0,0 +1,147 @@
+# PHP Events
+
+## Conversation related events
+
+### Created conversation
+
+* Event class: `OCA\Talk\Events\RoomEvent`
+* Event name: `OCA\Talk\Room::createdRoom`
```

Comment:
IMO it should be `roomCreated`. That sounds more like a description of an event. `createdRoom` is the action.